src/ostree/ot-builtin-checkout.c

```diff
@@ -293,37 +290,33 @@ process_many_checkouts (OstreeRepo *repo, const char *target, GCancellable *canc
 
       /* Read the null byte */
       (void)g_data_input_stream_read_byte (datastream, cancellable, NULL);
-      g_free (subpath);
```

Comment:
A  good example of why C99 is better, declare when you need helps prevent silliness like this.